test/clj/rems/test_browser.clj

```diff
@@ -270,8 +284,9 @@
     (testing "create application"
       (go-to-catalogue)
       (add-to-cart "Default workflow")
+      (add-to-cart "Private form workflow")
       (btu/gather-axe-results)
-      (apply-for-resource "Default workflow")
+      (apply-for-resource-bundle)
```

Comment:
`apply-for-resource` is left unused by this change (according to grep). i can remove that piece of code or leave it be - it served as backbone for the bundle version

Reply:
renamed